script/bootstrap.sh

```diff
@@ -99,12 +72,19 @@ if [ $(uname) = "Linux" ]; then
                          clang-format-6.0 \
                          cmake \
                          ninja-build \
+                         python-setuptools \
+                         python-pip \
                          lcov -y
 
-    ## Install cmake 3.16.1
+    ## Install newest CMake
+    match_version $(cmake --version | egrep -o '[0-9].*') ${MIN_CMAKE_VERSION} || {
+        pip install -U pip
```

Comment:
The integration test uses OT Commissioner THCI which requires python2 by default. We here stick to python2 and see if we can upgrade to python3 in a future PR.